### PR TITLE
Bump Application Services to v0.30.0.

### PR DESCRIPTION
**Pull request by eoger**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

**Comment by jonalmeida** [comment]
@eoger waiting for your sign-off here as well - local testing seems to work fine.

**Comment by eoger** [comment]
- Send Tab keys are re-created without any issue.
- After upgrade Send Tab works again.
- History and bookmarks syncs.
:shipit:

**Comment by grigoryk** [comment]
Thanks for testing this thoroughly!

File: buildSrc/src/main/java/Dependencies.kt [hunk 1]
```diff
@@ -25,7 +25,7 @@ object Versions {
     const val jna = "5.2.0"
     const val disklrucache = "2.0.2"
 
-    const val mozilla_appservices = "0.29.0"
+    const val mozilla_appservices  = "0.30.0"
```
Comment (jonalmeida) [inline buildSrc/src/main/java/Dependencies.kt]: ktlint is complaining about that extra space added before the equal sign. 😅

**Pull request merged**